.github/workflows/release.yml

```diff
@@ -70,3 +65,44 @@ jobs:
         run: |
           npm install
           npx semantic-release
+
+  ghpages:
+    name: Deploy gh-pages
+    needs: release
+    runs-on: ubuntu-latest
+    if: (inputs.force_deploy_ghpages == true) || (github.ref == 'refs/heads/main') || startsWith(github.ref, 'refs/tags/v') || (github.ref == 'refs/heads/gh-pages-test')
+    steps:
+      # https://github.com/japgolly/setup-everything-scala
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0 # Needed for the release tag // `git fetch --tags` will also work
+      # - name: Setup Java and Scala
+      #   uses: olafurpg/setup-scala@v14
+      #   with:
+      #     java-version: adopt@1.11
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+      # - name: Setup Scala.JS
+      #   uses: japgolly/setup-scalajs@v1
+      # - name: Cache sbt
+      #   uses: coursier/cache-action@v6.3
+
+      # ### create build deploy gh-pages ###
+      # - name: sbt "docAll; siteAll"
+      #   run: sbt -v "docAll; siteAll"
+
+      - name: Build
+        run: |
+          npm install
```

Comment:
would it be better to use yarn instead of npm?

Reply:
I think it needs to be install

Reply:
it will work as well, but we have a [readme.md](https://github.com/hyperledger/identus-docs/blob/main/README.md?plain=1#L63) file where we use yarn
it's not critical